disco_aws_automation/disco_aws.py

```diff
@@ -162,6 +162,7 @@ def create_userdata(self, hostclass, owner, testing):
                 data["route_table_id_{0}".format(_)] = _tmp[_][0]
                 data["destination_cidr_block_{0}".format(_)] = _tmp[_][1]
         logging.debug("userdata: %s", data)
+        data["project_name"] = self.config("project_name")
```

Comment:
Whoops. I'm assuming should be fine to just swap the new line and the logging line.